unit-tests/live/safety/test-safety-preset-read-write-hwm-cmd.py

```diff
@@ -2,7 +2,7 @@
 # Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 
 #test:donotrun:!nightly
```

Comment:
Lets add `#test:donotrun` currently until we have D585S connected to LibCI (With a comment)